apps/site/components/Downloads/Release/InstallationMethodDropdown.tsx

```diff
@@ -7,11 +7,12 @@ import type { FC } from 'react';
 
 import { ReleaseContext } from '#site/providers/releaseProvider';
 import type { InstallationMethod } from '#site/types/release';
+
 import {
   nextItem,
   INSTALL_METHODS,
   parseCompat,
-} from '#site/util/downloadUtils';
+} from '../../../downloadUtils/index';
```

Comment:
You shouldn't need this change this, it should be automatically inferred.

Ditto for the rest

Reply:
You can keep it in the utils folder, as its own subdirectory